src/commands/chat/verification/verifyrules.ts

```diff
@@ -1,154 +1,34 @@
-                        return;
-                    }
-
-                    const roles: RoleData[] = [];
-
-                    for (const roleName of importedRule.roles) {
-                        const role = interaction.guild?.roles.cache.find((role) => role.name === roleName);
-
-                        if (!role) {
-                            await this.sendErrorEmbed(
-                                interaction,
-                                'Invalid Role',
-                                `The role "${roleName}" could not be found on this server.`
-                            );
-                            return;
-                        } else if (!role.editable) {
-                            await this.sendErrorEmbed(
-                                interaction,
-                                'Unable to Assign Role',
-                                `I do not have permission to assign the "${roleName}" role. Make sure I have the \`Manage Roles\` permission and that my role is placed above all roles that you want to assign.`
-                            );
-                            return;
-                        } else {
-                            roles.push({ id: role.id, name: role.name });
-                        }
-                    }
-                    const newRule: VerificationRule = {
-                        roles: roles,
-                        department: String(importedRule.department),
-                        matchType: String(importedRule.match),
-                        yearMatch: String(importedRule.yearMatch),
-                    };
-
-                    if (importedRule.year) newRule.year = Number(importedRule.year);
-
-                    newRules.push(newRule);
-                }
-            }
-
-            const config = await GuildConfigCache.fetchOrCreate(interaction.guild!.id);
-
-            config.verificationRules = {
-                baseYear: -1, // TODO: remove base year from code
-                rules: newRules,
-            };
-
-            await config.save();
-
-            const embed = new MessageEmbed().setColor('GREEN').setTitle('Verification Rules Updated Successfully')
-                .setDescription(`Verification is ${
-                config.enableVerification ? 'enabled' : `disabled. Enable it using ${inlineCode('/config')}`
-            }. [Create a ruleset.](https://sebot.sunnyzuo.com/)
-                ${codeBlock(serializeVerificationRules(config.verificationRules))}`);
+    async execute(interaction: Message | CommandInteraction): Promise<void> {
+        const config = await GuildConfigCache.fetchConfig(interaction.guild!.id);
 
-            await interaction.reply({ embeds: [embed] });
-        } else {
-            const config = await GuildConfigCache.fetchConfig(interaction.guild!.id);
+        const button = new MessageActionRow().addComponents(
+            new MessageButton().setCustomId(`verifyRules`).setLabel('Update rules').setStyle('PRIMARY')
```

Comment:
```suggestion
            new MessageButton().setCustomId(`verifyRules`).setLabel('Update Rules').setStyle('PRIMARY')
```

for consistency with our other buttons